### PR TITLE
Background color css

### PR DESCRIPTION
**Pull request by GaryRidgway**
This is a start for #1654. This does not contain full work for people or events just quite yet, as those were unfinished at the time of creation. these could be revisited.

To Test:

1. pull down branch, composer install, blt frontend, drush cr
2. blt dsa the demo site from dev. you will need these pages to then test all of the items necessary:
    https://default.local.drupal.uiowa.edu/kitchen-sink
    https://default.local.drupal.uiowa.edu/kitchen-sink-webform
    https://default.local.drupal.uiowa.edu/kitchen-sink-carousel
3. check all variations and make sure they appear to be in a proper styled state.

### CODE REVIEW

**Comment by joewhitsitt** [comment]
Just curious, why the separate partial?

**Comment by GaryRidgway** [comment]
> Just curious, why the separate partial?

On the issue you mentioned a separate partial as a catch all, I assumed this was meant for the background color overrides so that is was more clear. If I was wrong we can of course move all of this back in to the other scss file.

**Comment by joewhitsitt** [comment]
This might be affected by the re-organization being done in #1757. Marking as blocked for now.

**Comment by joewhitsitt** [comment]
@GaryRidgway now that #1757 got merged, can you update this PR to reflect the re-organization.

**Comment by GaryRidgway** [comment]
> @GaryRidgway now that #1757 got merged, can you update this PR to reflect the re-organization.

on it today

**Comment by joewhitsitt** [comment]
pulling down for testing

File: docroot/sites/default/blt.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ project:
 drush:
   aliases:
     local: self
-    remote: default.prod
+    remote: default.dev
```
Comment (joewhitsitt) [inline docroot/sites/default/blt.yml]: We don't want to commit this

**Comment by GaryRidgway** [comment]
I have finally finished everything with the help of the kitchen sink pages. I would love if this could get merged for demo tomorrow but I also understand there is a lot to test. I've also updated the main PR comment with how to test if you are looking for some instructions.

**Comment by joewhitsitt** [comment]
General
bg dark - yellow links do not meet wcag 2.1 for contrast 4.5:1
Screen Shot 2020-06-25 at 9 39 49 AM


Webform
bg black - gold webform - the button text
Screen Shot 2020-06-25 at 9 45 14 AM

Carousel
default bg - bg light caption - link text
Screen Shot 2020-06-25 at 9 46 42 AM

**Review by joewhitsitt — APPROVED** [review]
Thanks for doing this tough work! My tests check out.

**Pull request merged**